src/reducers/index.js

```diff
@@ -98,6 +100,28 @@ const initialState = {
   startWalletAction: null,
   // RouterHistory object
   routerHistory: null,
+
+  // Should we allow using the atomic swap feature?
```

Comment:
Maybe change this comment to a docstring (`/**`)

Reply:
Done on [a893cb2](https://github.com/HathorNetwork/hathor-wallet/pull/361/commits/a893cb2ac1c6e746a5efdf352e5e81dd2d215f40)